### ISSUE TITLE
[HIGH submit form] Missing functionality on select a11y features

### ISSUE THREAD

**Issue by SteveALee** [w3c/wai-authoring-tools-list#97]
looking at the code

https://github.com/w3c/wai-authoring-tools-list/blob/master/_includes/report-feature-support.liquid#L24-L36

it seems the last three radio buttons for each feature should reveal some elements.

This is not working. Is it required?
